.tekton/operator-bundle-pipeline.yaml

```diff
@@ -660,3 +663,129 @@ spec:
       - name: kind
         value: task
       resolver: bundles
+
+  - name: create-acs-style-snapshot
+    # Explicitly running after all other tasks to ensure that
+    # - there are no failures that should prevent a release of the operator-bundle image (missing RPMs signatures, deprecated base images, ...)
+    # - the source image is present as it is required by EC
+    runAfter:
```

Comment:
I'd suggest you mention in this comment that `scripts/ci/jobs/check-konflux-pipelines.sh` as an easy way to keep `runAfter` list up-to-date.